<launch>
<arg name="deviceIndex" default="0"/>
<arg name="baudNum" default="1"/>
<node pkg="usb2ax_controller" type="ax_joint_controller" name="ax_joint_controller" args="$(arg deviceIndex) $(arg baudNum)" output="screen"/>
</launch>
